app/models/mynewlikes_new.rb

```diff
@@ -1,12 +1,12 @@
 class MynewlikesNew < ApplicationRecord
-    self.table_name = 'mynewlikes'
+  self.table_name = 'mynewlikes'
 
-    belongs_to :user, class_name: 'User'
-    belongs_to :post, class_name: 'MynewpostNew'
-  
-    after_save :update_like_counter
-  
-    def update_like_counter
-      post.update(like_counter: post.likes.count)
-    end
+  belongs_to :user, class_name: 'User'
+  belongs_to :post, class_name: 'MynewpostNew'
+
+  after_save :update_like_counter
+
+  def update_like_counter
+    post.update(like_counter: post.likes.count)
+  end
```

Comment:
- [ ] Kindly include tests for the custom method in your `Like Model` as well. I have highlighted the method above. So kindly include tests for it :wink: